Convert-ProwlarrSupportedIndexersToMarkdownTable.ps1

```diff
@@ -1,108 +1,150 @@
+### Private Torrents
+Write-Information 'Building: Torrents - Private'
+$tbl_PrvTor = $indexer_tbl_obj | Where-Object { ($_.privacy -CIn 'private' -and $_.protocol -eq 'torrent') } | Select-Object @{Name = 'Indexer'; Expression = $indexer_name_exp }, @{Name = 'Language'; Expression = { $_.language } }, @{Name = 'Description'; Expression = $indexer_description_exp }
+
+## Convert Data to Markdown Table
+$tbl_fmt_PubUse = $tbl_PubUse | Format-MarkdownTableTableStyle Indexer, Description, Language -HideStandardOutput -ShowMarkdown -DoNotCopyToClipboard
+$tbl_fmt_PrvUse = $tbl_PrvUse | Format-MarkdownTableTableStyle Indexer, Description, Language -HideStandardOutput -ShowMarkdown -DoNotCopyToClipboard
+$tbl_fmt_PubTor = $tbl_PubTor | Format-MarkdownTableTableStyle Indexer, Description, Language -HideStandardOutput -ShowMarkdown -DoNotCopyToClipboard 
+$tbl_fmt_PrvTor = $tbl_PrvTor | Format-MarkdownTableTableStyle Indexer, Description, Language -HideStandardOutput -ShowMarkdown -DoNotCopyToClipboard
+Write-Information 'Builds Converted to Markdown Tables'
+
+## Page Header Info
+$wiki_page_start = $wiki_1newline + "- Supported Indexers as of Build ``" + $build + "`` / [Commit: " + $commit + '](' + $wiki_commiturl + $commit + ')'
+Write-Information 'Page Header Built'
+
+## Build Page Pieces'
+$tbl_fmt_tor = $wiki_1newline + '## Torrents' + $wiki_2newline + '### Public Trackers' + $wiki_2newline + $tbl_fmt_PubTor + $wiki_1newline + '### Private Trackers' + $wiki_2newline + $tbl_fmt_PrvTor
+$tbl_fmt_use = $wiki_1newline + '## Usenet' + $wiki_2newline + '### Public Indexers' + $wiki_2newline + $tbl_fmt_PubUse + $wiki_1newline + '### Private Indexers' + $wiki_2newline + $tbl_fmt_PrvUse
+Write-Information 'Wiki Markdown Tables Built'
+$date = [DateTime]::UtcNow.ToString('o')
+$mdHeader = 
+"---
 title: Prowlarr Supported Indexers
-description: Indexers currently named as supported in the current nightly build of Prowlarr. Other indexers are available via either Generic Newznab or Generic Torznab. 
+description: Indexers currently named as supported in the current nightly build of Prowlarr. Other indexers are available via either Generic Newznab or Generic Torznab.
 published: true
 date: $date
 tags: prowlarr, indexers
 editor: markdown
 dateCreated: $date
 ---"
-
-$fmttableall= $mdHeader + "`n" + $prefix + "`n" + $fmttabletor + $fmttableuse
-$fmttableall | Out-File $OutputFile
+Write-Information 'Wiki Page pieces built'
+## Build and Output Page
+## We replace because converting to markdown escaped the `\` as `\\` and thus `\\\\` is `\\` in file (due to regex)
+$wiki_page_file = $mdHeader + $wiki_1newline + $wiki_page_start + $wiki_1newline + ($tbl_fmt_tor -replace '\\\\', '\') + ($tbl_fmt_use -replace '\\\\', '\') + $wiki_1newline 
```

Comment:
We could simplify this by having the whole thing in brackets and the replace on the whole string as opposed to individual elements assuming that there are no legitimate need for //// in the markdown!


Alternatively do it at the creation of the markdown variables and not in this variable creation

Reply:
The markdown function is where the issue occurs.

the data is clean going into the formats table and coming out it escapes everything.

Reply:
fixed and pushed